test/metabase/upload_test.clj

```diff
@@ -1115,6 +1118,47 @@
                                              :upload-seconds    pos?}}}
                    (last-audit-event :upload-create)))))))))
 
+(defn- write-1-pixel-jpg
```

Comment:
I was too lazy to add a test resource file, which felt like overkill. I could be coaxed.

Reply:
Is there nothing smaller that would pose as a binary file? 🤣 I'd say if it's more than 20-30 bytes it should go in resources. :)

Reply:
Hmm a 1-pixel GIF is 35b... I need to rise to your challenge 😆

Reply:
https://news.ycombinator.com/item?id=38878480

Reply:
Didn't need to be an image 😈